crategen/converters/tes_converter.py

```diff
@@ -1,20 +1,32 @@
+"""Module for converting TES data to WRROC format and vice versa."""
+
+from typing import Any, Dict
+
 from .abstract_converter import AbstractConverter
 from .utils import convert_to_iso8601
 
+
 class TESConverter(AbstractConverter):
+    """Converter for TES data to WRROC and vice versa."""
+
+    def convert_to_wrroc(self, tes_data: Dict[str, Any]) -> Dict[str, Any]:
+        """Convert TES data to WRROC format.
```

Comment:
Noted! I will refactor the code to eliminate the immediate return of inline variables. Additionally, I will rename the variable `id` to avoid shadowing the built-in Python identifier. Thanks for pointing this out!

Reply:
Again, remember to keep this PR to linting/style changes only. If you want to do a refactoring PR, that's fine/good, but it should be in a separate PR.